pkg/apis/openebs.io/v1alpha1/cstor_pool.go

```diff
@@ -37,10 +37,18 @@ type CStorPool struct {
 
 // CStorPoolSpec is the spec listing fields for a CStorPool resource.
 type CStorPoolSpec struct {
-	Disks    DiskAttr      `json:"disks"`
+	Group    []DiskGroup      `json:"group"`
```

Comment:
Can we have 
```go
DiskGroups []DiskGroup `json:"diskGroups"`
```

Reply:
It is according to design. And if it is appropriate I can change it. 
Let us discuss in design doc and then get it changed if this is what we want.